docs/conf/portland/2022/news/tickets-on-sale.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/generic.html
+
+.. post:: Feb 2, 2022
+   :tags: portland-2021, tickets
+
+
+Announcing ticket sales, full conference website, and CFP extension
+===================================================================
+
+Greeting, documentarians! Following our announcement that the Portland conference will remain virtual only, we're excited to announce that tickets are on sale and the full website is now live.
+
+Following the success of our previous virtual conferences, we look forward to more opportunities to bring the community together virtually and to share the wonderful energy with each other wherever you are in the world.
+
+Get your ticket for the conference
+----------------------------------
+
+**The Portland conference will be held online on {{ date.short }}.** The `full event website <https://www.writethedocs.org/conf/portland/{{year}}/>`_ is live and already includes some preliminary details. We'll be updating it with more information as we get closer to the event. 
+
+If you're excited too, go ahead and `register for your ticket now <https://www.writethedocs.org/conf/portland/{{year}}/tickets/>`_! We aren't sure exactly what capacity we will be able to hold for the virtual event, but we might still sell out to make sure we're providing a manageable event size for our attendees, as always.
+
+Similar to last year's virtual conference, we're going to keep as much of the spirit of the event as possible:
+
+* Excellent single track of pre-recorded presentations with moderated Q&A for each talk
+* Writing Day
+* Unconference sessions
+* Job Fair
+* Sponsor booths
+* A virtual hallway track
+* A social event of some kind, bring your own beverage and snacks :)
+
+We really want to make sure that folks can get value from being at the event at the same time, and we think it will be another great year of helping our community connect.
```

Comment:
```suggestion
We plan the event to ensure lots of interaction, which is the main reason we're all gathered together, and we think it will be another great year of helping our community connect.
```